feat: update ch 9 of bg (#977)
flybywiresim/docs: docs/pilots-corner/a380x/a380x-beginner-guide/09_powering-down.md

<link rel="stylesheet" href="/stylesheets/bg.css">

# Powering Down

This guide will explain the correct procedures to power down the aircraft when at the gate after arriving at the destination and taxiing to the designated gate.

Obviously this is not strictly required in a simulator, but for interested sim pilots this might be an interesting process for a more realistic experience.

!!! warning "Disclaimer"

>     The level of detail in this guide is meant to help a FlyByWire A380X beginner correctly shut down the aircraft.

Reviewer: ```suggestion
    The level of detail in this guide is meant to help a Airbus A380 beginner correctly shut down the aircraft.
```

- `APU` has been turned on during taxi and is `AVAIL`, `APU BLEED` is off.
- **After Landing** checklist is completed.

`RWY TURN OFF LTS ...................................................... ON`<br/>
`AUXILIARY POWER UNIT ............................................... AVAIL`<br/>
`NOSE LT ............................................................. TAXI`<br/>
`LANDING LTS .......................................................... OFF`<br/>

**At the gate:**

![EDDM at gate](../assets/beginner-guide/powering-down/eddm-at-the-gate.png "EDDM at gate"){loading=lazy} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

**Steps after arriving at the gate:**

> - Set parking brake (`PARK BRK`).

Reviewer: you can remove this - the following SOP line covers this - same for the other paragraphs below.

`PARKING BRAKE ......................................................... ON`<br/>

![Parking Brake](../assets/beginner-guide/powering-down/parking-brk.png "Parking Brake"){loading=lazy} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

> - `NOSE TAXI` and `RWY TURN OFF` lights are usually turned off even before we turn into the gate to not blind the ground personnel. This of course is only done if enough lighting is available to safely navigate and park into the gate. Turn them off if they were used while parking.

Reviewer: note admon under the next SOP line

`EXTERIOR LIGHTS ...................................................... OFF`<br/>

> ![Taxilights off](../assets/beginner-guide/powering-down/lights-taxi-off.png "Taxilights off"){loading=lazy} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

Reviewer: admon tip below the section "How and Where"

> - Turn on `APU BLEED` before we turn off the engines.

Reviewer: ```suggestion
```

`AUXILIARY POWER UNIT BLEED ............................................ ON`<br/>

> ![APU BLEED ON](../assets/beginner-guide/powering-down/apu-bleed-on1.png "APU BLEED ON"){loading=lazy width=20%} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

Reviewer: admon tip below the section "How and Where"

> - Shut down the engines by setting the `ENG 1`, `ENG 2`, `ENG 3` and `ENG 4` master switches to off.

Reviewer: ```suggestion
```

`ENGINE MASTER SWITCHES (1, 2, 3, 4) .................................. OFF`<br/>

> ![Engines off](../assets/beginner-guide/powering-down/engine-off.png "Engines off"){loading=lazy } <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

Reviewer: admon tip below the section "How and Where"

> - Wait until `N1` is below 5 %.

Reviewer: Could be a SOP line
```suggestion
`ENGINE N1 < 5%  .................................. OFF`<br/>
```

![Engines N1 under 5 %](../assets/beginner-guide/powering-down/ENG-N1-u5.png "Engines N1 under 5 %"){loading=lazy width=50%} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> <p style="color:yellow; font-size:18px;">is this necessary to keep in? not mentioned in SOP at all.</p>

> - Turn off `Seat Belt` sign.

Reviewer: ```suggestion
```

`SEAT BELTS ........................................................... OFF`<br/>

> ![Seat Belt Sign off](../assets/beginner-guide/powering-down/seatbelts-off.png "Seat Belt Sign off"){loading=lazy width=20%} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

Reviewer: admon tip below the section "How and Where"

- Turn off `BEACON` (leave `NAV & LOGO` on as long as the aircraft has power from external or APU, `STROBES` can remain on AUTO).

`BEACON ............................................................... OFF`<br/>

> ![All_Ext_LT_off](../assets/beginner-guide/powering-down/beacons-off.png){width=20%} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

Reviewer: admon tip below the section "How and Where"

> - Turn off all fuel pumps.

Reviewer: ```suggestion
```

`FUEL PUMPS ........................................................... OFF`<br/>

![Fuel pumps off](../assets/beginner-guide/powering-down/fuel-pumps-off.png "Fuel pumps off"){loading=lazy width=80%} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

> - Turn off the `ADIRS`.

Reviewer: ```suggestion
```

`ADIRS (1+2+3) ........................................................ OFF`<br/>

> ![ADIRS OFF](../assets/beginner-guide/powering-down/adirs-off.png "ADIRS OFF"){loading=lazy } <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

Reviewer: admon tip below the section "How and Where"

> - Turn off `OXYGEN`.

Reviewer: ```suggestion
```

`OXYGEN CREW SUPPLY ................................................... OFF`<br/>

![OXYGEN Crew Supply off](../assets/beginner-guide/powering-down/crewoxyhighlight.png "OXYGEN Crew Supply off"){loading=lazy} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

> - Turn off `APU BLEED`.

Reviewer: ```suggestion
```

`AUXILIARY POWER UNIT BLEED ........................................... OFF`<br/>

![APU BLEED OFF](../assets/beginner-guide/powering-down/apu-bleed.png "APU BLEED OFF"){loading=lazy width=20%} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

> - Turn off emergency exit lights `EMER EXIT LT` and no smoking lights `NO SMOKING`.

Reviewer: ```suggestion
```

`EMERGENCY EXIT LIGHTS ................................................ OFF`<br/>
`NO SMOKING ........................................................... OFF`<br/>

> ![Signs off](../assets/beginner-guide/powering-down/passenger-signs.png "Signs off"){loading=lazy} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

Reviewer: admon tip below the section "How and Where"

> - Optional or depending on airline SOPs: Reset air conditioning, lighting, and screen brightness.

Reviewer: SOP line

> - Turn off `APU MASTER` (expect the APU to still be AVAIL for a few minutes if you also had APU Bleed on shortly before, as it needs a cool down period).

Reviewer: ```suggestion
```

`AUXILIARY POWER UNIT MASTER SWITCH ................................... OFF`<br/>

> ![APU Master off](../assets/beginner-guide/powering-down/apu-master-off1.png "APU Master off"){loading=lazy width=20%} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

Reviewer: admon tip below the section "How and Where"

> - Wait 2 minutes for the APU FLAP door to close before you turn off the power, as this requires either APU or external power.

Reviewer: not sure this is required for the A380

> - Turn off `EXT PWR` if it has been turned on before

Reviewer: ```suggestion
```

`EXTERNAL POWER ....................................................... OFF`<br/>

> ![External Power off](../assets/beginner-guide/powering-down/ext-power-off.png "External Power off"){loading=lazy width=20%} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

Reviewer: admon tip below the section "How and Where"

> - Turn off `NAV & LOGO` lights (as the aircraft no longer has power).

Reviewer: ```suggestion
```

`EXTERIOR LIGHTS ...................................................... OFF`<br/>

> ![NAV & LOGO off](../assets/beginner-guide/powering-down/nav-logo-off-1.png "NAV & LOGO off"){loading=lazy width=20%} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

Reviewer: admon tip below the section "How and Where"

> - Turn off `BAT 1` and `BAT 2`.

Reviewer: ```suggestion
```

> `ALL BATT ............................................................. OFF`<br/>

Reviewer: Did you actually use the A380 SOP? I have not checked you page against it but it looks as if you simply rewrote the A320.
I think it is better to do these guides from ground up using the A380 SOP.
See this example:
![image](https://github.com/flybywiresim/docs/assets/16833201/9e7eaebc-cc57-4366-b1b4-2cd7d678d11c)

This would then look like this:
```suggestion
`ALL BATT (Battery 1, Essential, Battery 2, APU Battery) . . .. . . . . . . . . . . . . . . . . OFF`<br/>
??? note "APU Flaps"
	It is recommended to wait until the APU flap is closed before shutting off the APU battery.
```

Author: Yup, everything in chapter 9 came straight out of the SOP. No guesswork involved

> ![Batteries off](../assets/beginner-guide/powering-down/batteries-off.png "Batteries off"){loading=lazy width=70%} <p style="color:yellow; font-size:18px;">TODO: update screenshot</p> 

Reviewer: admon tip below the section "How and Where"
